homeassistant/components/mqtt/discovery.py

```diff
@@ -41,6 +41,114 @@
+    'osc_val_tpl':         'oscillation_value_template',
+    'pl_arm_away':         'payload_arm_away',
+    'pl_arm_home':         'payload_arm_home',
+    'pl_avail':            'payload_available',
+    'pl_cls':              'payload_close',
+    'pl_disarm':           'payload_disarm',
+    'pl_hi_spd':           'payload_high_speed',
+    'pl_lock':             'payload_lock',
+    'pl_lo_spd':           'payload_low_speed',
+    'pl_med_spd':          'payload_medium_speed',
+    'pl_not_avail':        'payload_not_available',
+    'pl_off':              'payload_off',
+    'pl_on':               'payload_on',
+    'pl_open':             'payload_open',
+    'pl_osc_off':          'payload_oscillation_off',
+    'pl_osc_on':           'payload_oscillation_on',
+    'pl_stop':             'payload_stop',
+    'pl_unlk':             'payload_unlock',
+    'pow_cmd_t':           'power_command_topic',
+    'ret':                 'retain',
+    'rgb_cmd_tpl':         'rgb_command_template',
+    'rgb_cmd_t':           'rgb_command_topic',
+    'rgb_stat_t':          'rgb_state_topic',
+    'rgb_val_tpl':         'rgb_value_template',
+    'send_if_off':         'send_if_off',
+    'set_pos_tpl':         'set_position_template',
+    'set_pos_t':           'set_position_topic',
+    'spd_cmd_t':           'speed_command_topic',
+    'spd_stat_t':          'speed_state_topic',
+    'spd_val_tpl':         'speed_value_template',
+    'spds':                'speeds',
+    'stat_clsd':           'state_closed',
+    'stat_off':            'state_off',
+    'stat_on':             'state_on',
+    'stat_open':           'state_open',
+    'stat_t':              'state_topic',
+    'stat_val_tpl':        'state_value_template',
+    'swing_mode_cmd_t':    'swing_mode_command_topic',
+    'swing_mode_stat_tpl': 'swing_mode_state_template',
+    'swing_mode_stat_t':   'swing_mode_state_topic',
+    'temp_cmd_t':          'temperature_command_topic',
+    'temp_stat_tpl':       'temperature_state_template',
+    'temp_stat_t':         'temperature_state_topic',
+    'tilt_clsd_val':       'tilt_closed_value',
+    'tilt_cmd_t':          'tilt_command_topic',
+    'tilt_inv_stat':       'tilt_invert_state',
+    'tilt_max':            'tilt_max',
+    'tilt_min':            'tilt_min',
+    'tilt_opnd_val':       'tilt_opened_value',
+    'tilt_status_opt':     'tilt_status_optimistic',
+    'tilt_status_t':       'tilt_status_topic',
+    't':                   'topic',
+    'uniq_id':             'unique_id',
+    'unit_of_meas':        'unit_of_measurement',
+    'val_tpl':             'value_template',
+    'whit_val_cmd_t':      'white_value_command_topic',
+    'whit_val_stat_t':     'white_value_state_topic',
+    'whit_val_tpl':        'white_value_template',
+    'xy_cmd_t':            'xy_command_topic',
+    'xy_stat_t':           'xy_state_topic',
+    'xy_val_tpl':          'xy_value_template',
```

Comment:
Please use only one whitespace between the key and the value. If one want to add a new long topic then all entries need to be changes to be aligned.

Reply:
I had it like that first, and it makes the list completely unreadable.
Maybe that's OK though since the list is also in the documentation?